src/components/frame/v5/pages/FundsPage/partials/FundsTable/FundsTable.tsx

```diff
@@ -0,0 +1,59 @@
+import {
+  getFilteredRowModel,
+  getPaginationRowModel,
+} from '@tanstack/react-table';
+import React, { FC } from 'react';
+import { useColonyFundsClaims } from '~hooks';
+import { formatText } from '~utils/intl';
+import EmptyContent from '~v5/common/EmptyContent';
+import TableWithActionsHeader from '~v5/common/TableWithActionsHeader';
+import AcceptButton from '../AcceptButton';
+import Filter from '../Filter';
+import { useFundsTable, useFundsTableColumns } from './hooks';
+
+const FundsTable: FC = () => {
+  const columns = useFundsTableColumns();
+  const { filters, searchedTokens } = useFundsTable();
+  const claims = useColonyFundsClaims();
+  const allClaims = Array.from(
+    new Set(claims.map((claim) => claim.token?.tokenAddress || '')),
+  );
+
+  return (
+    <TableWithActionsHeader
+      title={formatText({ id: 'incomingFundsPage.table.title' })}
+      data={searchedTokens}
+      columns={columns}
+      verticalOnMobile={false}
+      hasPagination
+      initialState={{
+        pagination: {
+          pageSize: 4,
+        },
+      }}
+      getFilteredRowModel={getFilteredRowModel()}
+      getPaginationRowModel={getPaginationRowModel()}
+      className="[&_td]:border-b [&_td]:border-gray-100 [&_tr:last-child>td]:border-0 [&_td]:p-0 rounded-t-none w-full"
+      emptyContent={
+        !searchedTokens.length && (
```

Comment:
Is this `emptyContent` different everytime you're using this table? If not, then maybe it would be good to just add it here directly to TableWithActionsHeader component?

Reply:
Texts will be different